service-deployment/src/grafana/prom-datasource.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name":"PM",
+  "type":"prometheus",
+  "url":"<PROM_URL>",
```

Comment:
>PROM_URL [](start = 10, length = 8)

what's this? from template?

Reply:
Yes, it's template. It will be replaced when running start.sh

Reply:
if so, prom-datasource.json should be prom-datasource.json.template, right? Please check w/ Yundong.

---
In reply to: [157502829](https://github.com/Microsoft/pai/pull/102#discussion_r157502829) [](ancestors = 157502829)